plugin.go

```diff
@@ -142,12 +126,17 @@ func (p *Plugin) installAddonViaHelm(release *types.Release) error {
 	cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeRaw, Value: release.ChartPath})
 	cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeLongParam, Name: "version", Value: release.Version})
 
+	if p.ClusterConfig.Helm.Debug {
+		cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeRaw, Value: "--debug"})
+	}
+
 	// Add namespaces to command
 	if release.Namespace != "" {
 		cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeLongParam, Name: "namespace", Value: release.Namespace})
 	}
-	if p.ClusterConfig.Helm.Namespace != "" {
-		cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeLongParam, Name: "tiller-namespace", Value: p.ClusterConfig.Helm.Namespace})
+
+	if p.ClusterConfig.Helm.LogLevel != "" {
```

Comment:
See comment below about making this a `uint`. You can then check the condition `> 0`.

Reply:
if i use Integer, i would have to convert it into string for `commandbuilder`, would it be easier to keep it as string instead of adding conversion?

Reply:
i see your point, having `uint` type will make sure log level is set to a number vs char